javascript/javascript3/week1/Weather/script.js

```diff
@@ -0,0 +1,44 @@
+//Weather API call
+const APIKEY = "9469946e53df5e8174588db17f99cd64";
+const limit = 3;
+
+const country = document.getElementById('search-country');
+const cityName = document.getElementById('search-city');
+const button = document.getElementById('button');
+
+const resultUl = document.getElementById('result-ul');
```

Comment:
Nitpick(naming):
Maybe something like `searchButton` would give more context.
Try also to avoid naming variables and CSS classes based on the details of the HTML markup.
`resultList` would be better, you could change it to use `div` or whatever and it's still a list.